.github/workflows/run-clang-format.sh

```diff
@@ -16,7 +16,7 @@
 FORMAT_OUT=${TMPDIR:-/tmp}/clang-format-diff.out
 
 # Run on all the files that are affected
```

Comment:
Maybe add a `clang-format --version` here?

Reply:
Looks like this is done in an earlier step....

Reply:
`clang-format version 10.0.0-4ubuntu1`

Reply:
```
  --dump-config              - Dump configuration options to stdout and exit.
                               Can be used with -style option.
  --fallback-style=<string>  - The name of the predefined style used as a
                               fallback in case clang-format is invoked with
                               -style=file, but can not find the .clang-format
                               file to use.
                               Use -fallback-style=none to skip formatting.
```
and
```
  --verbose                  - If set, shows the list of processed files
```
Could be useful too....

Reply:
`--fallback-style` I tried. It then does not fall back, which indicates that it found the file, but then didn't act properly.
`strace` was not installed, but that will be one of my next experiments in `testing-clang-format` to see if it picked up the file.